### DISCUSSION TITLE
Why did you remove the Celery and flower 3 month ago? what do you use as task queue now? thx

### DISCUSSION THREAD

**Discussion by tianerphil** [fastapi/full-stack-fastapi-template#1237] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
no code related
```

### Description

I am try to build a simple full stack web application to generate image. the task are long lasting so I think a task queue tool like celery + redis + flower are typical. but I found you have removed them out from your repo 3 month ago, I wonder you are using something else?

### Operating System

Linux

### Operating System Details

ubuntu

### Python Version

3.10.14

**Comment by elliotblackburn** [comment, marked as answer]
I'd also be keen to understand this a bit more. The documentation points to this template as an example of using Celery with Fast API (https://fastapi.tiangolo.com/tutorial/background-tasks/?h=celery#caveat). I was a bit confused when I realised it wasn't here at all. I'm coming from a different language ecosystem so would be really keen to understand if it's being dropped in favour of another recommendation, or if it was just hard to maintain in the template.

**Comment by AlejandroBlanco2001** [comment]
@tianerphil Maybe this [PR](https://github.com/tiangolo/full-stack-fastapi-template/pull/694#issue-2182357172) can give you some more context

**Comment by tianerphil** [comment]
Understood. Thank you for giving these context.


While just update you that I have add celery worker, celery beat, redis, flower into the fleet. I have to do so, Since the task meant to be very time consuming.&nbsp;


So far so good, It do increased the complexity of debugging, but the tools are convenient to do so, docker and flower could give the necessary debugging information in 98% cases.




Will Let you know more when I put them for pressure test.


Thx









Best&nbsp;Regards,
Liang&nbsp;ZHANG
0509119456

---Original---
From: "Isaac ***@***.***&gt;
Date: Thu, Jul 4, 2024 17:44 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [tiangolo/full-stack-fastapi-template] Why did you remove theCelery and flower 3 month ago? what do you use as task queue now? thx(Discussion #1237)




 
@tianerphil Maybe this PR can give you some more context
